jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -2705,6 +2705,397 @@ YUI.add('juju-env-api', function(Y) {
     },
 
     /**
+      Return information about all entities in this model.
+
+      @param {Function} callback A callable that must be called once the
+        operation is performed. It will receive two arguments: an error string
+        and an empty object if an error occurred, null and the status result
+        otherwise. The status result is an object similar to the following:
+
+          {
```

Comment:
This information is basically a duplication of the return object in the function. Why not just add the comments inline there if it's ambiguous?